### ISSUE TITLE
Request - "Remember me" / persistent login

### ISSUE THREAD

**Issue by raysguy** [Hellowlol/HTPC-Manager#119]
With HTPC I seem to find myself having to login almost every time I want to take a quick look and it really interrupts my workflow.  I don't have the same problem with Sonarr, CP, SickRage etc as their login pages feature a "remember me" option, so on my own computer I can type in the address and see the dash straight away, even between browsing sessions.  Is that something that could be easily implemented?  Thanks.

**Comment by Hellowlol** [comment]
I have incresed session timeout for noe.

**Comment by raysguy** [comment]
Thanks, I'll see how it goes.  I'm doing a lot of restarting of HTPC at the moment whilst testing so having to login quite a big anyway.

**Comment by Hellowlol** [comment]
Yeah, the session is kept in memory so all users need to auth again. There no builtin remember me in cherrypy so i have to make it myself and it requires some small db changes.
